web/mysql_typemap.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2020 Google LLC
```

Comment:
Is there are test for this?
Similar comment for postgres_typemap.go?

Reply:
No, there are no separate tests for this, as this is just providing a helper function in my point of view and is just switch case statements, let me know if i should write tests for these.

Reply:
Isn't it tested in web_test.go?

Reply:
Yes, they do test the functionality of mysql_typemap.go as part of TestUpdateTableSchemaPostgres, TestUpdateTableSchemaMySQL, TestSetTypeMapGlobalLevelMySQL and TestSetTypeMapGlobalLevelPostgres. but there are no separate test for the function defined in mysql_typemap.go or postgrs_typemap.go

Reply:
Let's put a comment here to indicate this.
In general, we should have a test for every file.
However, in some cases it doesn't make sense to do this because the code is extensively tested elsewhere as part of another chunk of code. Suggest a comment at the top of the function something like:

// toSpannerTypeMySQL defines the mapping of source types into Spanner
// types. Each source type has a default Spanner type, as well as other potential
// Spanner types it could map to. When calling toSpannerTypeMySQL, you specify
// the source type name (along with any modifiers), and optionally you specify
// a target Spanner type name (empty string if you don't have one). If the target
// Spanner type name is specified and is a potential mapping for this source type,
// then it will be used to build the returned ddl.Type. If not, the default
// Spanner type for this source type will be used.
// Note that toSpannerTypeMySQL is extensively tested via tests in web_test.go.

Let's put a similar comment in web/postgres_typemap.go.